### PR TITLE
Allow SSH to IMPAIRED clusters

### PR DESCRIPTION
**Pull request by thesquelched**
Allow SSH to impaired clusters.  Also do some housekeeping, e.g. creating release script, adding some missing docs, and fixing the `--version` flag.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
+0.2.9
+-----
+* CLI
+    * Fixed --version flag
+* You may now SSH to any cluster not in ERROR status, including IMPAIRED
```
Comment (jimbobhickville) [inline CHANGELOG.md]: It looks like you can SSH to ERROR as well, just not any in-flight status like BUILDING, CONFIGURING, or UPDATING

Comment (thesquelched) [inline CHANGELOG.md]: Should be good now @jimbobhickville, and I added tests

**Comment by jimbobhickville** [comment]
+1

**Comment by amock** [comment]
+1

**Pull request merged**